build.sh

```diff
@@ -390,8 +415,10 @@ if [ ${BUILD_BE} -eq 1 ] ; then
                   -DWITH_STARCACHE=${WITH_STARCACHE}                    \
                   -DUSE_STAROS=${USE_STAROS}                            \
                   -DENABLE_FAULT_INJECTION=${ENABLE_FAULT_INJECTION}    \
+                  -DCMAKE_EXPORT_COMPILE_COMMANDS=ON  ..                \
```

Comment:
it's evil to hide the `..` among those `-D` options.

Reply:
fixed